manifest.json

```diff
@@ -39,6 +39,7 @@
   "settingsSchema": {
     "title": "admin/store.title",
     "type": "object",
+    "bindingBounded": true,
```

Comment:
Can you explain the impact of this flag?

Reply:
Not sure I understood your question, but this flag is an opt in flag that allows the store to have different settings per binding. If the store doesn't turn it on, there is no impact.